[0m09:49:03.375125 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F22C3EA2D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F230AE5490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F230AE69D0>]}


============================== 09:49:03.378204 | 69a75cf2-7a5d-41bc-8768-69bc1cd3a11a ==============================
[0m09:49:03.378204 [info ] [MainThread]: Running with dbt=1.10.4
[0m09:49:03.379210 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init Telegram_Med_pipeline', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m09:49:03.391909 [debug] [MainThread]: Starter project path: C:\Program Files\Python311\Lib\site-packages\dbt\include\starter_project
[0m09:49:03.399705 [info ] [MainThread]: 
Your new dbt project "Telegram_Med_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:49:03.400849 [info ] [MainThread]: Setting up your profile.
[0m09:56:34.983696 [info ] [MainThread]: Profile Telegram_Med_pipeline written to C:\Users\hause\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m09:56:34.989783 [debug] [MainThread]: Command `dbt init` succeeded at 09:56:34.989783 after 451.71 seconds
[0m09:56:34.993288 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F22EA6BA50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F230A6AA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F228D3A1D0>]}
[0m09:56:34.996871 [debug] [MainThread]: Flushing usage events
[0m09:56:36.405082 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:08:47.970601 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D22E9A310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D262FBC90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D263042D0>]}


============================== 10:08:47.970601 | 6fc33759-ac8f-4218-9533-0c0302cac49f ==============================
[0m10:08:47.970601 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:08:47.970601 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'invocation_command': 'dbt init', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m10:09:02.311494 [info ] [MainThread]: A project called Telegram_Med_pipeline already exists here.
[0m10:09:02.314542 [debug] [MainThread]: Command `dbt init` succeeded at 10:09:02.314542 after 14.43 seconds
[0m10:09:02.314542 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2752AAD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D2756B150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024D263041D0>]}
[0m10:09:02.317129 [debug] [MainThread]: Flushing usage events
[0m10:09:12.611631 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:12:57.808958 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E337A38D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E324FBE10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E3371D5D0>]}


============================== 10:12:57.811464 | a5cb162e-cf13-4a48-85dd-0d6f645bfdf2 ==============================
[0m10:12:57.811464 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:12:57.812734 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt seed --full-refresh', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m10:12:57.813941 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m10:12:57.813941 [debug] [MainThread]: Command `dbt seed` failed at 10:12:57.813941 after 0.10 seconds
[0m10:12:57.813941 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E3372B3D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E3372A990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011E337A2B90>]}
[0m10:12:57.815938 [debug] [MainThread]: Flushing usage events
[0m10:12:59.351943 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:14:49.770823 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B038BA390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B06D1BC90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B07F4AC10>]}


============================== 10:14:49.775922 | 2e3df4b7-8c1e-415e-9c17-ad71f15b3023 ==============================
[0m10:14:49.775922 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:14:49.777008 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m10:15:04.163138 [debug] [MainThread]: Starter project path: C:\Program Files\Python311\Lib\site-packages\dbt\include\starter_project
[0m10:15:04.198501 [info ] [MainThread]: 
Your new dbt project "Telegram_Med_pipeline_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:15:04.198501 [info ] [MainThread]: Setting up your profile.
[0m10:16:18.901717 [error] [MainThread]: Encountered an error:

[0m10:16:18.921384 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Program Files\Python311\Lib\site-packages\dbt\cli\requires.py", line 161, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\site-packages\dbt\cli\requires.py", line 111, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\site-packages\dbt\cli\main.py", line 483, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\site-packages\dbt\task\init.py", line 346, in run
    self.setup_profile(profile_name)
  File "C:\Program Files\Python311\Lib\site-packages\dbt\task\init.py", line 261, in setup_profile
    adapter = self.ask_for_adapter_choice()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\site-packages\dbt\task\init.py", line 242, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hause\AppData\Roaming\Python\Python311\site-packages\click\termui.py", line 168, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hause\AppData\Roaming\Python\Python311\site-packages\click\termui.py", line 151, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m10:16:18.932436 [debug] [MainThread]: Command `dbt init` failed at 10:16:18.930906 after 89.25 seconds
[0m10:16:18.933329 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B07FC1F90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B07F4A910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025B07F8B390>]}
[0m10:16:18.933329 [debug] [MainThread]: Flushing usage events
[0m10:16:20.258170 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:35:21.788885 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189FAD32B10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189F75F7910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189F7619110>]}


============================== 10:35:21.792061 | b8e1f616-80da-41d7-8e9a-8292bd624453 ==============================
[0m10:35:21.792061 [info ] [MainThread]: Running with dbt=1.10.4
[0m10:35:21.792980 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m10:35:21.804980 [info ] [MainThread]: dbt version: 1.10.4
[0m10:35:21.805980 [info ] [MainThread]: python version: 3.11.0
[0m10:35:21.805980 [info ] [MainThread]: python path: C:\Program Files\Python311\python.exe
[0m10:35:21.807307 [info ] [MainThread]: os info: Windows-10-10.0.22631-SP0
[0m10:35:21.856878 [info ] [MainThread]: Using profiles dir at C:\Users\hause\.dbt
[0m10:35:21.857877 [info ] [MainThread]: Using profiles.yml file at C:\Users\hause\.dbt\profiles.yml
[0m10:35:21.858797 [info ] [MainThread]: Using dbt_project.yml file at D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml
[0m10:35:21.859797 [info ] [MainThread]: adapter type: postgres
[0m10:35:21.860797 [info ] [MainThread]: adapter version: 1.9.0
[0m10:35:21.860797 [info ] [MainThread]: Configuration:
[0m10:35:21.861818 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m10:35:21.862797 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m10:35:21.862797 [info ] [MainThread]: Required dependencies:
[0m10:35:21.863798 [debug] [MainThread]: Executing "git --help"
[0m10:35:21.907014 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m10:35:21.907014 [debug] [MainThread]: STDERR: "b''"
[0m10:35:21.907979 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m10:35:21.908979 [info ] [MainThread]: Connection:
[0m10:35:21.908979 [info ] [MainThread]:   host: localhost
[0m10:35:21.909980 [info ] [MainThread]:   port: 5432
[0m10:35:21.910978 [info ] [MainThread]:   user: postgres
[0m10:35:21.910978 [info ] [MainThread]:   database: telegram
[0m10:35:21.911977 [info ] [MainThread]:   schema: raw
[0m10:35:21.912976 [info ] [MainThread]:   connect_timeout: 10
[0m10:35:21.912976 [info ] [MainThread]:   role: None
[0m10:35:21.913977 [info ] [MainThread]:   search_path: None
[0m10:35:21.913977 [info ] [MainThread]:   keepalives_idle: 0
[0m10:35:21.914977 [info ] [MainThread]:   sslmode: None
[0m10:35:21.915982 [info ] [MainThread]:   sslcert: None
[0m10:35:21.915982 [info ] [MainThread]:   sslkey: None
[0m10:35:21.916977 [info ] [MainThread]:   sslrootcert: None
[0m10:35:21.917980 [info ] [MainThread]:   application_name: dbt
[0m10:35:21.918980 [info ] [MainThread]:   retries: 1
[0m10:35:21.919979 [info ] [MainThread]: Registered adapter: postgres=1.9.0
[0m10:35:22.095496 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m10:35:25.311320 [debug] [MainThread]: Using postgres connection "debug"
[0m10:35:25.311320 [debug] [MainThread]: On debug: select 1 as id
[0m10:35:25.311320 [debug] [MainThread]: Opening a new connection, currently in state init
[0m10:35:25.370507 [debug] [MainThread]: SQL status: SELECT 1 in 0.061 seconds
[0m10:35:25.370507 [debug] [MainThread]: On debug: Close
[0m10:35:25.370507 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m10:35:25.370507 [info ] [MainThread]: [31m1 check failed:[0m
[0m10:35:25.378553 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml> not found

[0m10:35:25.378553 [debug] [MainThread]: Command `dbt debug` failed at 10:35:25.378553 after 3.68 seconds
[0m10:35:25.378553 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m10:35:25.378553 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189FACBA9D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189FACBAE10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000189FBFDC450>]}
[0m10:35:25.378553 [debug] [MainThread]: Flushing usage events
[0m10:35:26.565369 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m12:53:05.028157 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AED195E650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AED196B490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AED19E3CD0>]}


============================== 12:53:05.037233 | 588f00d3-2d36-40a8-9af7-ac5bb214b806 ==============================
[0m12:53:05.037233 [info ] [MainThread]: Running with dbt=1.10.4
[0m12:53:05.037233 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:53:14.478522 [info ] [MainThread]: A project called Telegram_Med_pipeline already exists here.
[0m12:53:14.480527 [debug] [MainThread]: Command `dbt init` succeeded at 12:53:14.480527 after 9.55 seconds
[0m12:53:14.482527 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AED19E3150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AED1A11790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AEC9C39950>]}
[0m12:53:14.483527 [debug] [MainThread]: Flushing usage events
[0m12:53:15.540555 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m12:53:42.098534 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BAF261890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BB3E44F90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BB50680D0>]}


============================== 12:53:42.101949 | 39680aea-8ce0-4201-8555-1e7983eee6cf ==============================
[0m12:53:42.101949 [info ] [MainThread]: Running with dbt=1.10.4
[0m12:53:42.101949 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m12:53:45.379221 [debug] [MainThread]: Starter project path: C:\Program Files\Python311\Lib\site-packages\dbt\include\starter_project
[0m12:53:45.433491 [info ] [MainThread]: 
Your new dbt project "Telegram_Med_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:53:45.434042 [info ] [MainThread]: Setting up your profile.
[0m12:54:59.892681 [info ] [MainThread]: Profile Telegram_Med_pipeline written to C:\Users\hause\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m12:54:59.895679 [debug] [MainThread]: Command `dbt init` succeeded at 12:54:59.894680 after 77.88 seconds
[0m12:54:59.895679 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BB3E44290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BAD329950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020BAD5C3AD0>]}
[0m12:54:59.898226 [debug] [MainThread]: Flushing usage events
[0m12:55:00.901730 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m12:55:22.764390 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF74120A90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF740AB5D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF741001D0>]}


============================== 12:55:22.767896 | 221a4868-ad00-49fd-a622-d8864c38c8b4 ==============================
[0m12:55:22.767896 [info ] [MainThread]: Running with dbt=1.10.4
[0m12:55:22.769224 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:55:22.788069 [info ] [MainThread]: dbt version: 1.10.4
[0m12:55:22.789069 [info ] [MainThread]: python version: 3.11.0
[0m12:55:22.790071 [info ] [MainThread]: python path: C:\Program Files\Python311\python.exe
[0m12:55:22.790071 [info ] [MainThread]: os info: Windows-10-10.0.22631-SP0
[0m12:55:22.832878 [info ] [MainThread]: Using profiles dir at C:\Users\hause\.dbt
[0m12:55:22.832878 [info ] [MainThread]: Using profiles.yml file at C:\Users\hause\.dbt\profiles.yml
[0m12:55:22.833965 [info ] [MainThread]: Using dbt_project.yml file at D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml
[0m12:55:22.839069 [info ] [MainThread]: adapter type: postgres
[0m12:55:22.840235 [info ] [MainThread]: adapter version: 1.9.0
[0m12:55:22.840235 [info ] [MainThread]: Configuration:
[0m12:55:22.841670 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m12:55:22.841670 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m12:55:22.843074 [info ] [MainThread]: Required dependencies:
[0m12:55:22.843074 [debug] [MainThread]: Executing "git --help"
[0m12:55:22.889351 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m12:55:22.890579 [debug] [MainThread]: STDERR: "b''"
[0m12:55:22.890579 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m12:55:22.891583 [info ] [MainThread]: Connection:
[0m12:55:22.892610 [info ] [MainThread]:   host: localhost
[0m12:55:22.892610 [info ] [MainThread]:   port: 5432
[0m12:55:22.892610 [info ] [MainThread]:   user: postgres
[0m12:55:22.892610 [info ] [MainThread]:   database: telegram
[0m12:55:22.893795 [info ] [MainThread]:   schema: raw
[0m12:55:22.893795 [info ] [MainThread]:   connect_timeout: 10
[0m12:55:22.893795 [info ] [MainThread]:   role: None
[0m12:55:22.894922 [info ] [MainThread]:   search_path: None
[0m12:55:22.894922 [info ] [MainThread]:   keepalives_idle: 0
[0m12:55:22.894922 [info ] [MainThread]:   sslmode: None
[0m12:55:22.895925 [info ] [MainThread]:   sslcert: None
[0m12:55:22.895925 [info ] [MainThread]:   sslkey: None
[0m12:55:22.895925 [info ] [MainThread]:   sslrootcert: None
[0m12:55:22.895925 [info ] [MainThread]:   application_name: dbt
[0m12:55:22.897432 [info ] [MainThread]:   retries: 1
[0m12:55:22.898450 [info ] [MainThread]: Registered adapter: postgres=1.9.0
[0m12:55:23.071756 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m12:55:23.499855 [debug] [MainThread]: Using postgres connection "debug"
[0m12:55:23.500860 [debug] [MainThread]: On debug: select 1 as id
[0m12:55:23.500860 [debug] [MainThread]: Opening a new connection, currently in state init
[0m12:55:23.541583 [debug] [MainThread]: SQL status: SELECT 1 in 0.040 seconds
[0m12:55:23.542555 [debug] [MainThread]: On debug: Close
[0m12:55:23.542555 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m12:55:23.543523 [info ] [MainThread]: [31m1 check failed:[0m
[0m12:55:23.544522 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml> not found

[0m12:55:23.545105 [debug] [MainThread]: Command `dbt debug` failed at 12:55:23.545105 after 0.88 seconds
[0m12:55:23.546211 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m12:55:23.546211 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF740AB3D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF74121F90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AF75290BD0>]}
[0m12:55:23.546211 [debug] [MainThread]: Flushing usage events
[0m12:55:24.621427 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m12:58:17.575416 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C337EBE090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C33C5B0950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C33C5B2B90>]}


============================== 12:58:17.577950 | 27b88025-d731-4188-9bf0-c860b290019d ==============================
[0m12:58:17.577950 [info ] [MainThread]: Running with dbt=1.10.4
[0m12:58:17.577950 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'target_path': 'None', 'invocation_command': 'dbt test', 'send_anonymous_usage_stats': 'True'}
[0m12:58:17.579419 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:58:17.580832 [debug] [MainThread]: Command `dbt test` failed at 12:58:17.580832 after 0.09 seconds
[0m12:58:17.580832 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C33C5B1E10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C334AF3A50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C33C53B450>]}
[0m12:58:17.581835 [debug] [MainThread]: Flushing usage events
[0m12:58:18.648790 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m13:43:00.915776 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F509D1E10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F509D1050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F5094DB50>]}


============================== 13:43:00.918370 | 3360f1a3-35f4-4dee-91db-26ffedc157ec ==============================
[0m13:43:00.918370 [info ] [MainThread]: Running with dbt=1.10.4
[0m13:43:00.919393 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\hause\\.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m13:43:00.932007 [info ] [MainThread]: dbt version: 1.10.4
[0m13:43:00.932949 [info ] [MainThread]: python version: 3.11.0
[0m13:43:00.933486 [info ] [MainThread]: python path: C:\Program Files\Python311\python.exe
[0m13:43:00.933486 [info ] [MainThread]: os info: Windows-10-10.0.22631-SP0
[0m13:43:00.979862 [info ] [MainThread]: Using profiles dir at C:\Users\hause\.dbt
[0m13:43:00.980852 [info ] [MainThread]: Using profiles.yml file at C:\Users\hause\.dbt\profiles.yml
[0m13:43:00.980852 [info ] [MainThread]: Using dbt_project.yml file at D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml
[0m13:43:00.981856 [info ] [MainThread]: adapter type: postgres
[0m13:43:00.982980 [info ] [MainThread]: adapter version: 1.9.0
[0m13:43:00.982980 [info ] [MainThread]: Configuration:
[0m13:43:00.984272 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m13:43:00.984272 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m13:43:00.985278 [info ] [MainThread]: Required dependencies:
[0m13:43:00.986275 [debug] [MainThread]: Executing "git --help"
[0m13:43:01.036837 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m13:43:01.038403 [debug] [MainThread]: STDERR: "b''"
[0m13:43:01.038403 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m13:43:01.039524 [info ] [MainThread]: Connection:
[0m13:43:01.039524 [info ] [MainThread]:   host: localhost
[0m13:43:01.040800 [info ] [MainThread]:   port: 5432
[0m13:43:01.042802 [info ] [MainThread]:   user: postgres
[0m13:43:01.044802 [info ] [MainThread]:   database: telegram
[0m13:43:01.045800 [info ] [MainThread]:   schema: raw
[0m13:43:01.046802 [info ] [MainThread]:   connect_timeout: 10
[0m13:43:01.046802 [info ] [MainThread]:   role: None
[0m13:43:01.048308 [info ] [MainThread]:   search_path: None
[0m13:43:01.048308 [info ] [MainThread]:   keepalives_idle: 0
[0m13:43:01.049362 [info ] [MainThread]:   sslmode: None
[0m13:43:01.049362 [info ] [MainThread]:   sslcert: None
[0m13:43:01.050669 [info ] [MainThread]:   sslkey: None
[0m13:43:01.050669 [info ] [MainThread]:   sslrootcert: None
[0m13:43:01.051668 [info ] [MainThread]:   application_name: dbt
[0m13:43:01.052673 [info ] [MainThread]:   retries: 1
[0m13:43:01.052673 [info ] [MainThread]: Registered adapter: postgres=1.9.0
[0m13:43:01.215951 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m13:43:01.256743 [debug] [MainThread]: Using postgres connection "debug"
[0m13:43:01.256743 [debug] [MainThread]: On debug: select 1 as id
[0m13:43:01.258347 [debug] [MainThread]: Opening a new connection, currently in state init
[0m13:43:01.306503 [debug] [MainThread]: SQL status: SELECT 1 in 0.049 seconds
[0m13:43:01.307504 [debug] [MainThread]: On debug: Close
[0m13:43:01.309009 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m13:43:01.310012 [info ] [MainThread]: [31m1 check failed:[0m
[0m13:43:01.311017 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\KAIM 5,6,7\Week-7\Shipping-Data-Product\dbt_project.yml> not found

[0m13:43:01.313016 [debug] [MainThread]: Command `dbt debug` failed at 13:43:01.312016 after 0.49 seconds
[0m13:43:01.313016 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m13:43:01.313016 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F5095B350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F509B5450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000023F4FAC3C10>]}
[0m13:43:01.314015 [debug] [MainThread]: Flushing usage events
[0m13:43:02.480451 [debug] [MainThread]: An error was encountered while trying to flush usage events
